examples/main.tf

```diff
@@ -108,6 +131,15 @@ resource "instaclustr_vpc_peering" "example_vpc_peering" {
   peer_subnets = toset(["10.0.0.0/20", "10.0.32.0/20"])
 }
 
+resource "instaclustr_vpc_peering_gcp" "example_vpc_peering" {
+
+  name="name"
+  peer_vpc_network_name = "network name"
+  peer_project_id = "projectId"
+  peer_subnets = toset(["10.10.0.0/16", "10.11.0.0/16"])
+  cluster_id = "${instaclustr_cluster.gcp_example.id}"
```

Comment:
**[Question]** Do you need to wait for the cluster to provision to create a VPC peering connection?

Reply:
Yup but we can also straight away create a peering to an already provisioned cluster by having the cluster id inside `cluster_id`

Reply:
Right, so shouldn't the example resources for peering have the `wait_for_state=RUNNING` field enabled if you have both a cluster and VPC in one resource and provisioned via the same plan?

Reply:
Done